releases/nightly-build/compose-files/docker-compose-nexus-add-device-services.yml

```diff
@@ -0,0 +1,122 @@
+#  *
+#  * @author: Leonard Goodell
+#  * EdgeX Foundry, Hanoi, version "master"
+#  * added: May, 2020
+#  *******************************************************************************/
+
+# NOTE:  this Docker Compose file does not contain the security services - namely the API Gateway
+# and Secret Store
+
+version: '3.7'
+
+# all common shared environment variables defined here:
+x-common-env-variables: &common-variables
+  REGISTRY_HOST: edgex-core-consul
+  CLIENTS_COREDATA_HOST: edgex-core-data
+  CLIENTS_NOTIFICATIONS_HOST: edgex-support-notifications
+  CLIENTS_METADATA_HOST: edgex-core-metadata
+  CLIENTS_COMMAND_HOST: edgex-core-command
+
+services:
+  device-virtual:
+    image: ${REPOSITORY}/docker-device-virtual-go${ARCH}:master
+    ports:
+    - "127.0.0.1:49990:49990"
+    container_name: edgex-device-virtual
+    hostname: edgex-device-virtual
+    networks:
+      - edgex-network
+    environment:
+      <<: *common-variables
+      SERVICE_HOST: edgex-device-virtual
+    depends_on:
+      - consul
+      - data
+      - metadata
+
+  device-rest:
+    image: ${REPOSITORY}/docker-device-rest-go${ARCH}:master
+    ports:
+      - "127.0.0.1:49986:49986"
+    container_name: edgex-device-rest
+    hostname: edgex-device-rest
+    networks:
+      - edgex-network
+    environment:
+      <<: *common-variables
+      SERVICE_HOST: edgex-device-rest
+    depends_on:
+      - data
+      - command
+
+#  device-random:
+#    image: ${REPOSITORY}/docker-device-random-go${ARCH}:master
+#    ports:
+#      - "127.0.0.1:49988:49988"
+#    container_name: edgex-device-random
+#    hostname: edgex-device-random
+#    networks:
+#      - edgex-network
+#    environment:
+#      <<: *common-variables
+#      SERVICE_HOST: edgex-device-random
+#    depends_on:
+#      - data
+#      - command
+#
+#  device-mqtt:
+#    image: ${REPOSITORY}/docker-device-mqtt-go${ARCH}:master
+#    ports:
+#      - "127.0.0.1:49982:49982"
+#    container_name: edgex-device-mqtt
+#    hostname: edgex-device-mqtt
+#    networks:
+#      - edgex-network
+#    environment:
+#      <<: *common-variables
+#      SERVICE_HOST: edgex-device-mqtt
+#    depends_on:
+#      - data
+#      - command
+#
+#  device-modbus:
+#    image: ${REPOSITORY}/docker-device-modbus-go${ARCH}:master
+#    ports:
+#      - "127.0.0.1:49991:49991"
+#    container_name: edgex-device-modbus
+#    hostname: edgex-device-modbus
+#    networks:
+#      - edgex-network
+#    environment:
+#      <<: *common-variables
+#      SERVICE_HOST: edgex-device-modbus
+#    depends_on:
+#      - data
+#      - command
+#
+#  device-snmp:
+#    image: ${REPOSITORY}/docker-device-snmp-go${ARCH}:master
+#    ports:
+#      - "127.0.0.1:49993:49993"
+#    container_name: edgex-device-snmp
+#    hostname: edgex-device-snmp
+#    networks:
+#      - edgex-network
+#    environment:
+#      <<: *common-variables
+#      SERVICE_HOST: edgex-device-snmp
+#    depends_on:
+#      - data
+#      - command
```

Comment:
anyway for this to be handled more elegantly?

Reply:
Handle what more elegantly?

Reply:
The commented out services... ;-)

Reply:
yeah exactly that. Rubber ducking: could these by moved into their own compose files? Not a blocker just talking out loud